test/dispatcher.js

```diff
@@ -62,7 +75,7 @@ suite('Event Generation and Dispatching', function() {
 
   test('PointerEvents from mouse fire anywhere by default', function() {
     // move always fires
-    var cb = chai.spy();
+    var cb = chai.spy;
```

Comment:
What changed about `chai.spy` that this seems to have been a factory before, but no longer is?